### PR TITLE
FLUID-6262: Adding missing namespaces to prefs framework.

### PR DESCRIPTION
**Pull request by jobara**
https://issues.fluidproject.org/browse/FLUID-6262

### CODE REVIEW

**Comment by cindyli** [comment]
Is this missed:
- https://github.com/jobara/infusion/blob/FLUID-6262/src/framework/preferences/js/Panels.js#L650

Thinking if we should take the chance to namespace distribute options too.

**Comment by jobara** [comment]
@cindyli I added namespaces to the distributeOptions blocks. Ready for more review.

File: src/components/uiOptions/js/UIOptions.js [hunk 1]
```diff
@@ -21,24 +21,31 @@ var fluid_3_0_0 = fluid_3_0_0 || {};
...
-            source: "{that}.options.tocTemplate",
-            target: "{that uiEnhancer}.options.tocTemplate"
-        }, {
-            source: "{that}.options.ignoreForToC",
-            target: "{that uiEnhancer}.options.ignoreForToC"
-        }],
-        enhancer: {
-            distributeOptions: [{
+        distributeOptions: {
+            "uio.separatedPanel.azyLoad": {
```
Comment (cindyli) [inline src/components/uiOptions/js/UIOptions.js]: "azyLoad" -> "lazyLoad"? Missing "l".

**Comment by jobara** [comment]
@cindyli fixed the typo, ready for more review.

**Comment by cindyli** [comment]
Merged at 0d498cf2f3e08fe74927226ac9426e577c8d7682

**Pull request merged**